nextflow.config

```diff
@@ -105,12 +111,12 @@ dag {
 
 manifest {
   name = 'nf-core/scrnaseq'
-  author = 'Peter J Bailey, Alexander Peltzer'
+  author = 'Peter J Bailey, Alexander Peltzer, Olga Botvinnik'
   homePage = 'https://github.com/nf-core/scrnaseq'
   description = 'Pipeline for processing of 10xGenomics single cell rnaseq data'
   mainScript = 'main.nf'
-  nextflowVersion = '>=0.32.0'
-  version = '1.0dev'
+  nextflowVersion = '>=19.10.0'
```

Comment:
Maybe we just have 19.10. everywhere - will turn off TravisCI now that we have the better Github Actions Ci anyways. Just can't delete the `.travis.yml` yet, because nf-core lint will only get support for Github Actions with V1.8

Reply:
This is now consistent everywhere.